distributions/Dist_ejt352.py

```diff
@@ -1,21 +1,26 @@
 import numpy as np
 from .base_distribution import BaseDistribution
 
+
 class Dist_ejt352(BaseDistribution):
 	def __init__(self):
-		self.f_max = 1
-		self.x_min = -1
+		self.f_max = (12/24)
+		self.x_min = 0
 		self.x_max = 1
 
 
 	def pdf(self, x):
 		"""This is your PDF"""
-		return np.abs(x)
+		return (x**5)+(7/8)*x**2
+        """This function from 0 to 1 = (11/24)"""
```

Comment:
I think this is going to cause a python problem because the tabs aren't the same and it's after the return and not right after the `def pdf(self, x):` line. Can you delete this line or use a # at the beginning for a python comment

Reply:
best to make sure the tab lines up with return line.

Reply:
YEs, it causes a problem. See here:
https://travis-ci.com/github/cranmer/intro-exp-phys-II/builds/163581336#L257